docs/resources/auth_config_keycloak.md

```diff
@@ -41,6 +41,7 @@ The following arguments are supported:
 * `enabled` - (Optional) Enable auth config provider. Default `true` (bool)
 * `annotations` - (Optional/Computed) Annotations of the resource (map)
 * `labels` - (Optional/Computed) Labels of the resource (map)
+* `entity_id` - (Optional) KeyCloak Client ID field, defaults to `https://yourRancherHostURL/v1-saml/keycloak/saml/metadata` (string)
```

Comment:
Where is coming the default value for `entity_id` value??

Reply:
De "default" value is from Rancher itself. Now I am reconsidering, does that mean the value actually is `Optional/Computed` ?

Reply:
This may be one option. Define the argument as `Optional/Computed` without default value